django/core/checks/model_checks.py

```diff
@@ -25,6 +29,20 @@ def check_all_models(app_configs=None, **kwargs):
             )
         else:
             errors.extend(model.check(**kwargs))
+    duplicate_db_table_list = [
+        db_table_attr for db_table_attr, model_list in db_table_attr_map.items()
+        if len(model_list) > 1
+    ]
+    if duplicate_db_table_list:
+        for db_table in duplicate_db_table_list:
```

Comment:
It's not worth doing an existence check or a double list construction. Simply iterate and `continue`.

```python
for db_table, model_labels in db_tables.items():
    if len(model_labels) == 1:
        continue
    ...

Reply:
Yeah, that sounds good to me!